### DISCUSSION TITLE
Porting the windows community toolkit to UNO

### DISCUSSION THREAD

**Discussion by lucaregini** [unoplatform/uno#11580] [Q&A] [answered]
How much difficult would be to port the Windows Community Toolkit (https://github.com/CommunityToolkit/WindowsCommunityToolkit) to UNO? I am asking that because there are a couple of controls there that I would like to use and would like to understand if there is potential for migrating to UNO. In other words the question is whether UNO supports the same level of APIs and custom control can be easily migrated.

**Comment by FrozDark** [comment, marked as answer]
Ported https://github.com/unoplatform/Uno.WindowsCommunityToolkit

**Reply by jeromelaban** [reply]
You can find documentation here: https://platform.uno/docs/articles/uno-community-toolkit.html?tabs=tabid-winui
